py/embedded-server/deephaven_server/cli.py

```diff
@@ -22,21 +22,18 @@ def cli():
 @click.option("--port", default=None, type=int, help="The port to bind to.")
 @click.option("--jvm-args", default=None, help="The JVM arguments to use.")
 @click.option("--extra-classpath", default=None, help="The extra classpath to use.")
-def server(host, port, jvm_args, extra_classpath):
+@click.option("--default-jvm_args", default=None, help="The advanced JVM arguments to use in place of the default ones that Deephaven recommends.")
```

Comment:
Mixed `-` and `_`, on purpose?

Reply:
Thanks for catching it, a typo, not on purpose.